packages/shared/src/components/Feed.tsx

```diff
@@ -133,6 +133,7 @@ const getStyle = (useList: boolean, spaciness: Spaciness): CSSProperties => {
 const PostModalMap: Record<PostType, typeof ArticlePostModal> = {
   [PostType.Article]: ArticlePostModal,
   [PostType.Share]: SharePostModal,
+  [PostType.Welcome]: SharePostModal,
```

Comment:
Why do you add this if it's the same modal?
Not quite sure I get this one.

Reply:
This is to identify which modal the feed will use for the post clicked.

Reply:
But they all render the same modal?
Should we not rahter have a PostModal with prop type? (It seems a bit redundant to have 2 calls do the same modal.

Reply:
Ah sorry was misreading this piece, it's just the render that's the same.
For some reason thought you where introducing a new lazy modal with the same modal rendered 😅